HPCC-17317 DOCS: ECLWatch Add Delete Part

g-pan opened the pull request:
Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on 
using the Roxie Package Maps, Add, Delete, Activate Part.

Initial Pull Request, Expecting comments and feedback. 

@JamesDeFabia please review
@miguelvazq please review
@afishbeck please review 

Build results and PDF can be reviewed at 
http://10.240.32.243/job/DocDev0IV/3/CE=ubuntu-14.04-amd64-1/artifact/build-docs/Release/docs/The_ECL_Watch_Manual-6.3.0-trunk0.pdf
(Chrome users may need to download the file to review)

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [x] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [x] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 70274155df89527463a970e06eda91db587aaef3
Build: failed
Number of Error(s): 511
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid
 ... ( comment is too long, 518 lines, 87751 bytes, truncated) ... 
 Building CXX object aws-cpp-sdk-core/CMakeFiles/aws-cpp-sdk-core.dir/source/client/CoreErrors.cpp.o
[100%] Building CXX object aws-cpp-sdk-sqs/CMakeFiles/aws-cpp-sdk-sqs.dir/source/SQSErrors.cpp.o
make: *** [all] Error 2
Get ECLWatch build result


```

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
-        <para>You can select the <emphasis role="bold">Validate Package
-        Map</emphasis> tab to validate a package map.</para>
+        <sect3>
+          <title id="Packagemap_Actions">Packagemap Actions</title>
```
s/b two words: Package Map Actions

g-pan replied:
Don't think you are looking at the latest commit. This was fixed in a later commit.

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure>To Add a packagemap to the target cluster:</para>
```
package map (two words)

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
+
+            <listitem>
+              <para>Choose a Process FIlter /*what is command line
+              equiv?*/</para>
```
You left a comment. Replace with the description

afishbeck replied:
btw, I don't think I support process filter on command line.  It's one of those complex things that if I start to think people really want or need it I will add but over the last 5 years or so no one seems to be needing it.

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
+
+            <listitem>
+              <para>Check the boxes to Activate or Overwrite as
+              desired.</para>
```
Check the boxes to Activate or Overwrite, as desired

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
+          <para><emphasis role="bold">Deactivate Package Map</emphasis></para>
+
+          <para>Press the deactivate button to deactivate the currently active
+          package map.</para>
```
`<emphasis role="bold">Deactivate</emphasis>  button`

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
-        role="bold">Process</emphasis> from the drop lists on the Validate
-        Package Map tab.</para>
+          <para>Press the Activate button to deactivate the currently active
+          package map and make the selected package map active.</para>
```
`<emphasis role="bold">Activate</emphasis>  button`

JamesDeFabia reviewed (changes requested):
Comments inline

miguelvazq commented:
@g-pan Additional functionality for parts is located in the details as we discussed offline.

richardkchapman commented:
Needs rebasing foe a clean merge

afishbeck commented on docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml:
```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
-            <title>Validate Package Maps</title>
+          <para>To update package mappings, you would edit the packagemap file
+          (or a copy) add and activate the new one, and then delete the old
+          one. You can do this through the interface in ECLWatch.</para>
```
ECLWatch or command line?

afishbeck reviewed (commented):
@g-pan a few comments and general bits of knowledge.

richardkchapman commented:
Needs rebasing for a clean merge

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 60e5d9917a737a8a37e679dbf0d62815b088e67e
Build: failed
Number of Error(s): 85
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid 
 ... ( comment is too long, 92 lines, 13512 bytes, truncated) ... 
 Building CXX object aws-cpp-sdk-core/CMakeFiles/aws-cpp-sdk-core.dir/source/client/CoreErrors.cpp.o
[100%] Building CXX object aws-cpp-sdk-sqs/CMakeFiles/aws-cpp-sdk-sqs.dir/source/SQSErrors.cpp.o
make: *** [all] Error 2
Get ECLWatch build result


```

g-pan commented:
Keeping pull request open, while work in progress - preserving comments.  Will rebase when ready. Please hold off merging until rebased.

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 15efcfe260b67ba913d3b95825d1484569251077
Build: failed
Number of Error(s): 85
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10030/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid 
 ... ( comment is too long, 92 lines, 13512 bytes, truncated) ... 
 Building CXX object aws-cpp-sdk-core/CMakeFiles/aws-cpp-sdk-core.dir/source/client/CoreErrors.cpp.o
[100%] Building CXX object aws-cpp-sdk-sqs/CMakeFiles/aws-cpp-sdk-sqs.dir/source/SQSErrors.cpp.o
make: *** [all] Error 2
Get ECLWatch build result


```

g-pan commented:
Due in part to a hardware outage, and in part to some of the commit conflicts and comments here, I am closing this pull request, only to replace it with a single commit pull request.
